.github/workflows/chromatic.yml

```diff
@@ -20,7 +20,7 @@ permissions:
 
 jobs:
   chromatic-deployment:
-    runs-on: ubuntu-latest
+    runs-on: default_x64_latest
```

Comment:
We'll have to make sure these are switched back before merging as it's been fixed https://github.com/wellcomecollection/wellcomecollection.org/pull/12047